api/specs/web-server/_wallets.py

```diff
@@ -60,6 +60,14 @@ async def list_wallets():
     ...
 
 
+@router.get(
+    "/wallet",
```

Comment:
I think this is a good approach as soon as the default wallet always exists. I interpret this as a [singleton resouce](https://cloud.google.com/apis/design/design_patterns#singleton_resources) of the user, i.e. a user always have a wallet (no need to create/delete it), and I must have update and read operations.

An alternative I have been using would be to use the existing collection's entrypoint as `GET /wallets/default`
SEE https://github.com/ITISFoundation/osparc-simcore/blob/master/services/web/server/src/simcore_service_webserver/products/_handlers.py#L59C46-L59C46